wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,41 +1,57 @@
+  - The circle is not hit and its 50 hit window passes.
+
+### Sliders
+
+Sliders consist of multiple parts: the slider head, slider tail, slider ticks, and slider repeats. The slider as a whole is judged based on how many parts of the slider the player has hit, as outlined below:
+
+- 300: The player successfully hit all slider parts.
+- 100: The player hit at least half of the slider parts.
+- 50: The player managed to hit at least one of the parts.
+- Miss: The player did not successfully hit any of the slider parts.
+- If [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, the accuracy of hitting the slider head is additionally accounted for when judging the entire slider:
+  - Getting a 300 for a slider requires getting a 300 judgement for the slider head.
+  - Getting a 100 for a slider requires getting a 100 judgement or better for the slider head.
+
+The slider parts also have additional influence on the current [combo](/wiki/Gameplay/Combo_(score_multiplier)):
+
+- Tapping the slider head too early does not incur a miss, but will reset combo to 0.
+- Missing a slider tick or repeat does not incur a miss, but will reset combo to 0.
+- Missing the slider end does not incur a miss, but will not increment combo.
+
+### Spinners
+
+Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap. The criteria for judging the spinner as a whole are as follows:
 
-- A 300, 100 or 50 given from a normal hit circle depending on how accurately the player tap on it.
-- A miss given if the player do not click on the hit circle at all or click on it way too early.
+- 300: The player successfully performed the number of spins required for completion, or more.
+- 100: The player performed one spin less than the required number.
+- 50: The player performed at least 25% of the required spins.
+- Miss: The player failed to meet the minimum of 25% of spins required.
 
-**Sliders:**
+For some very short spinners, the number of spins required is actually calculated to be 0, and thus the spinner always automatically completes itself with a 300.
 
-- The judgement for sliders does not depend on the accuracy of the initial hit.
-- Sliders consist of slider ticks, which include the start, end and repeat points of the slider.
-- A 300 given if the player successfully gets all slider ticks, a 100 given if the player gets at least half of them, and a 50 given if the player manages to collect even one of the ticks.
-- Tapping a slider **too early** does not incur a miss but will **reset** the score multiplier to **0**.
-  - Not collecting the slider ticks, shown or otherwise, will not incur a miss, but will **reset** the score multiplier to **0**.
-  - Missing the slider end does not incur a miss but will **not** increment the score multiplier.
-- A miss given if the player collects none of the slider ticks.
+#### History
```

Comment:
```suggestion
## History
```

Reply:
I originally made it a subsection of the spinner section since that was the only place where history was relevant but sure, I guess no harm.

Reply:
i'm pretty sure the history section should basically always be a level 2 heading, since it also opens up for more history to be put there